examples/relay-fax.php

```diff
@@ -0,0 +1,48 @@
+<?php
+error_reporting(E_ALL);
+
+require dirname(__FILE__) . '/../vendor/autoload.php';
+
+$project = isset($_ENV['PROJECT']) ? $_ENV['PROJECT'] : '';
+$token = isset($_ENV['TOKEN']) ? $_ENV['TOKEN'] : '';
+if (empty($project) || empty($token)) {
+  throw new \Exception('Set your SignalWire project and token before run the example!');
+}
+
+$client = new SignalWire\Relay\Client(array(
+  "project" => $project,
+  "token" => $token
+));
+
+$client->on('signalwire.ready', function($session) {
+
+  $params = array('type' => 'phone', 'from' => '+1xxx', 'to' => '+1yyy');
+
+  $session->calling->dial($params)->done(function($dialResult) {
+    if (!$dialResult->isSuccessful()) {
+      echo "\n Error dialing \n";
+    }
+    $call = $dialResult->getCall();
+
+    $call->on('stateChange', function ($call) {
+      echo PHP_EOL . $call->id . " state changed to " . $call->state . PHP_EOL;
+    })
+    ->on('fax.stateChange', function ($call, $params) {
+      echo PHP_EOL . $call->id . " Fax Notification " . PHP_EOL;
+      print_r($params);
+    });
+
+    $call->faxSend('https://www.antennahouse.com/XSLsample/pdf-v32/Sample-multi-pdf3.pdf')->done(function($result) {
```

Comment:
Can we use a more official test PDF for the example: 

https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf
```suggestion
    $call->faxSend('https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf')->done(function($result) {
```